app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -282,6 +284,15 @@ private fun ConversationScreen(
         }
     }
 
+    val menuModalOnMessageDetailsClick = remember {
+        {
+            // conversationScreenState.hideEditContextMenu() // TODO: doesnt hide itself and some menus are hidden because its null
```

Comment:
This doesn't seem to be working when opening a new screen.

The ContextMenu/bottom sheet modal is still shown when coming back from open screen (MessageDetailsScreen) and its references are null, thus when clicking in any item shown it will result in the app crashing.

if you have any solution to this, please let me know

Reply:
This was resolved by adding a `onComplete` callback in `hideEditContextMenu` as it is using a coroutine scope, so when the corountine is done then we emit the callback making sure we won't be getting thrown a `CancellationException` from the animation of hiding the Context Menu.